src/sidebar/templates/login_control.html

```diff
@@ -20,7 +20,12 @@
   </a>
   <ul class="dropdown-menu pull-right" role="menu">
     <li class="dropdown-menu__row" ng-if="vm.auth.status === 'logged-in'">
-      <a href="{{vm.serviceUrl('user',{user: vm.auth.username})}}"
+      <a ng-if="!vm.isHAuthority()"
```

Comment:
If this text is not clickable it should just be a plain `<span>`. In this case, if the username is not clickable then the "View all your annotations" title doesn't make sense.